tests/routes/privacy.consent.test.ts

```
@@ -0,0 +1,310 @@
+          marketingOptout: true,
+          saleOptout: false,
+          biometricConsent: true,
+          createdAt: new Date("2026-07-25T10:00:00Z"),
+          updatedAt: new Date("2026-07-25T10:00:00Z"),
+        };
+
+        (prisma.privacyConsent.upsert as jest.Mock).mockResolvedValue(mockDbRecord);
+
+        const res = await request(app)
+          .put("/api/privacy/consent")
+          .send({
```

ESLint

[error] Replace `⏎··········.put("/api/privacy/consent")⏎··········` with `.put("/api/privacy/consent")`

(prettier/prettier)